### PR TITLE
Adds ability to set FADA pod name in torpedo flag

### PR DESCRIPTION
**Pull request by Pure-AdamuKaapan**
**What this PR does / why we need it**:
To test FADA multitenancy, it would be helpful to be able to add pod names to existing specs, instead of making whole separate apps just for those. This adds a variable to add the pod name into storage classes as we create them, in certain cases.

### CODE REVIEW

File: deployments/deploy-ssh.sh [hunk 1]
```diff
@@ -547,6 +551,7 @@ spec:
             "--pure-volumes=$IS_PURE_VOLUMES",
             "--pure-fa-snapshot-restore-to-many-test=$PURE_FA_CLONE_MANY_TEST",
             "--pure-san-type=$PURE_SAN_TYPE",
+            "--pure-fada-pod=$PURE_FADA_POD",
```
Comment (lsrinivas-pure) [inline deployments/deploy-ssh.sh:554]: @Pure-AdamuKaapan one quick question, will there be a scenario where we need to test multiple apps with different pod names?

**Review by vprabhakar-px — APPROVED** [review]
LGTM

**Review by vumesh-px — APPROVED** [review]
This is okay for having a single pod for all apps in the entire run. 
Had an offline conversation with @lsrinivas-pure  and @sayalasomayajula-px, we will also need a way for CustomAppConfig (which could be outside this PR)

File: deployments/deploy-ssh.sh (hunk 1 above)
Comment (Pure-AdamuKaapan) [reply deployments/deploy-ssh.sh:554]: Yes but we aren't tackling that yet, we can figure out the specifics later (as noted in the comment from Vishal)

**Review by pp511 — APPROVED** [review]

**Pull request merged**